ROADMAP.md

```diff
@@ -2,40 +2,47 @@
 
 ## User facing roadmap
 
-### V1beta1 support and GA
+### pipeline API v1 support
 
-- Add support for v1beta1 and later on GA
-- Warn usage of PipelineResource in v1beta1 (no error but warn that it is not supported aka will work but can be broken in a later release)
-- Support for workspace, result, …
+- Add support for v1 API when it will be publish
+- Support for fields, …
```

Comment:
Do we know which specific fields will need to be supported differently? If not it might just be fine to omit this point.